fw/http_msg.c

```diff
@@ -815,6 +820,33 @@ __hdr_sub(TfwHttpMsg *hm, const TfwStr *hdr, unsigned int hid)
 	return TFW_PASS;
 }
 
+/**
+ * Substitute request method with "get"
+ *
+ * Note: replaced method name length is supposed to be shorter
+ *       than the replacement string ("get")
+ */
```

Comment:
I am sorry but the comment sounds ambiguous and unclear. Suggestion: `the name length of the replaced method must be the same or longer than the length of the replacement string ("GET")` or `the name length of the method being replaced must be the same or longer than the length of the replacement string ("GET")`.

Reply:
ok. done.